### PR TITLE
Semaphore Identity example code bug fix (Fix #848)

### LINKED ISSUE #848: The sample code has bug(# Identity.verifySignature)
The sample code in the readme of Semaphore identity(# Identity.verifySignature) has reference error.
Because the "signature" pass in "signMessage" function didn't declare.

``` bash
ReferenceError: Cannot access 'signature' before initialization
```

### PR DESCRIPTION
**Pull request by csiejimmyliu**
**Description**
 
I deleted the undeclared input "signature" parameter in the `signMessage` function.

**Issue**

Closes #848

### CODE REVIEW

File: packages/identity/README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ import { Identity } from "@semaphore-protocol/identity"
 const message = "message"
 const identity = new Identity()
 
-const signature = identity.signMessage("message", signature)
+const signature = identity.signMessage("message")
```
Comment (vplasencia) [inline packages/identity/README.md]:
```suggestion
const signature = identity.signMessage(message)
```

What about using this for consistency?

**Review by vplasencia — CHANGES_REQUESTED** [review]
Thank you very much @csiejimmyliu

File: packages/identity/README.md (hunk 1 above)
Comment (vplasencia) [reply packages/identity/README.md]: Hey @csiejimmyliu! I just saw that you requested my review, did you push the changes?

Comment (csiejimmyliu) [reply packages/identity/README.md]: I pushed the changes. Thanks for the advice!

**Review by vplasencia — APPROVED** [review]

**Comment by csiejimmyliu** [comment]
Hey @vplasencia 
It says Merging is blocked(The base branch requires all commits to be signed.)
I follow some instructions, but I can only sign the newest commit.
Could you tell me how I can sign previous commits?
(BTW this is my first pull request XD)

**Comment by vplasencia** [comment]
Hey @csiejimmyliu no worries. It's fine. Great work 🚀

**Pull request merged**